packages/cache/README.md

```diff
@@ -42,3 +42,8 @@ const restoreKeys = [
 const cacheKey = await cache.restoreCache(paths, key, restoreKeys)
 ```
 
+##### Cache segment restore timeout
+
+Starting `v3.0.5` of `actions/toolkit`, in case any issue occurs while downloading the cache, the download will be aborted by default within 1 hour if any `segment` doesn't download completely. A `segment` is limited to size of `1GB` for a `32-bit` runner and `2GB` for a `64-bit` runner. So for any cache that exceeds the size of one segment, multiple segments will be downloaded in sequence to complete the download.
```

Comment:
Put the version of cache action as that is what is visible to users. Toolkit is an internal concept.

Reply:
I plan to put a similar readme in `actions/cache` where we can put the actions/cache version. I can remove the version from readme if its not needed. I just felt documenting the version in readme will be helpful for the readers of this readme file.

Reply:
Ah this is toolkit README. My bad. This is good. But we should add same section in cache action readme as well.

Reply:
Yes!